### ISSUE TITLE
Set of ops supported must be more comprehensive

### ISSUE THREAD

**Issue by anssiko** [w3c/machine-learning-charter#23]
(Related to the WebML WG Charter in development at https://github.com/w3c/machine-learning-charter/pull/19)

We [discussed](https://www.w3.org/2022/10/06-webmachinelearning-minutes.html#t04) "v2" use cases for WebNN and @wchao1115 shared the following feedback:

>… for v2, one of the constant feedback from our external partners when discussing WebNN for their use case has been the ops
>… the set of ops supported must be more comprehensive
>… this needs to be more explicit goal, this is important
>… related to that, use cases around transformers

The current charter [Scope](https://www.w3.org/2021/04/web-machine-learning-charter.html#scope) enumerates a few common ones: "convolution, pooling, softmax, normalization, fully connected, activation, recurrent neural network (RNN) and long short-term memory (LSTM)". This is not meant to be an all inclusive list and does give the WG ability to adapt to the changes in this landscape.

At minimum, we should review the bullets in the Scope section, and see whether to explicitly mention some of the more recent work such as transformers. We want to give enough detail to give good direction without constraining the WG too much. The list of ops mentioned in the charter would be open-ended.

**Comment by anssiko** [comment]
The WG's work is motivated by the compelling user experiences it enables to web users. To that end, I'm proposing to add the following informative text to the [Motivation and Background](https://www.w3.org/2021/04/web-machine-learning-charter.html#bg) section:

>Computer Vision enables computers to gain understanding from images or videos, Natural Language Processing enables interaction between computers and human languages, and Speech Recognition enables computers to recognize and translate spoken language into text. Bringing these experiences to the web in a privacy-preserving manner requires efficient machine learning inference capabilities built into the browser.

I think in addition we may want to clarify in the Scope section that the list of ops enumerated are examples of more established ops and the WG wants to give priority to ops that accelerate the above mentioned user experiences in CV, NLP, and Speech Recognition. Thus I'm proposing to add this non-binding text into the Scope section following the bullet list: 

>This Working Group puts priority on building blocks required by well-known model architectures in the fields of Computer Vision, Natural Language Processing and Speech Recognition."
